DEVELOPMENT.md

```diff
@@ -1,40 +1,80 @@
 # Development
 
-## Testing via examples
-
-1. Add a new example project in `examples` folder. It is a regular NPM package with its own `package.json` and Cypress dev dependency.
-1. Add a corresponding `.github/workflows` YAML file that uses this action and runs inside `examples/X` working directory. The example should demonstrate the feature
-1. Add workflow badge to README if needed
-
-## Testing against another repo
-
-1. Create a new local branch for any development, for example `git checkout -b featureA`
-1. Update the source code in [index.js](index.js)
-1. Build `dist` file(s) using `npm run build`
-1. Commit any changed files, note the SHA of the commit
-1. Push the local branch to GitHub
-1. In a test repository, create a test branch, change the action to point at the new commit. Example
-
-```yml
-name: End-to-end tests
-on: push
-jobs:
-  cypress-run:
-    runs-on: 22.04
-    steps:
-      - uses: actions/checkout@v3
-      # use specific commit pushed to GitHub
-      - uses: cypress-io/github-action@<full commit SHA>
+This document describes topics useful to contributors to this repository. The repository's purpose is to provide a GitHub JavaScript action which is published to:
+
+1. the npm JavaScript Package Registry as [@cypress/github-action](https://www.npmjs.com/package/@cypress/github-action)
```

Comment:
```suggestion
1. The npm JavaScript Package Registry as [@cypress/github-action](https://www.npmjs.com/package/@cypress/github-action)
```

Reply:
I used lower case because I was continuing the sentence which started before the list. Since this is a bit untidy I have take `the` out and used a possessive form instead.